CMakeLists.txt

```diff
@@ -22,12 +22,15 @@ find_package(XKBCommon REQUIRED)
 find_package(WLC REQUIRED)
 find_package(A2X REQUIRED)
 find_package(PCRE REQUIRED)
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(JSON REQUIRED json-c)
```

Comment:
You can use pkg-config inside the cmake file, I just want consistent dependency resolution between all of our dependencies.